contrib/drivers/scutech/cms/replication.go

```diff
@@ -0,0 +1,127 @@
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Please add the copyright header.

Reply:
add  # Copyright 2019 The OpenSDS Authors.

Reply:
add  # Copyright 2019 The OpenSDS Authors.